src/lib.rs

```diff
@@ -15,7 +15,7 @@
 //!
 //! |             crate | non-global | `'static` opt | non-`str` |  symbol size  | symbols deref
 //! | ----------------: | :--------: | :-----------: | :-------: | :-----------: | :-----------:
-//! |   simple-interner |    yes     |      no       |    yes    |     `&T`      |      yes
+//! |   simple-interner |    yes     |      yes[^1]  |    yes    |     `&T`      |      yes
```

Comment:
Actually, it'd be

```suggestion
//! |   simple-interner | optionally |      no        |    yes    |     `&T`      |      yes
```

The `'static` optimization is storing `&'static` references without copying the pointee into the backing store, e.g. storing `Cow<'static, str>` instead of `Box<str>` so you can track `&'static str`. We could add that and store some `enum MaybeStatic { Boxed(PinBox<T>), Static(&'static T) }`, but I'm not sure how beneficial it'd be.

Global refers to the ability to have a global interner that gives out `&'static`-live symbols.

It'd probably be good to clarify this. I should probably also split the first column into two, for "global" and "local" interner support, as that'd be clearer than the "optionally" entry.

Reply:
Thanks for explaining that! I've updated the table in f516a683df84ddd50369015dd9b933ccda8c50e1 to split the local vs global column, and corrected my `'static` opt field mistake.